steps/email.ts

```diff
@@ -1,21 +1,53 @@
+/* eslint-disable no-console */
 import { isAfter, parseISO, subMinutes } from 'date-fns'
 import { NotifyClient } from 'notifications-node-client'
 import { expect } from '@playwright/test'
 
 import { TestOptions } from '../testOptions'
 
-export const verifyEmailSent = async (subject: string, emailAddress: TestOptions['user']['email']) => {
+export const verifyEmailSent = async (
+  emailAddress: TestOptions['user']['email'],
+  subject: string,
+  bodyIncludes?: string | undefined,
+) => {
   await expect(async () => {
     const client = new NotifyClient(process.env.NOTIFY_API_KEY)
     const response = await client.getNotifications('email')
     const { notifications } = response.data
-    const matchedNotification = notifications.find(
-      notification =>
-        notificationWasSentInTheLastMinute(notification) &&
-        notification.subject === subject &&
-        notification.email_address === emailAddress,
+    const notificationsForEmail = notifications.filter(
+      notification => notificationWasSentInTheLastMinute(notification) && notification.email_address === emailAddress,
     )
-    expect(matchedNotification).toBeTruthy()
+
+    if (notificationsForEmail.length === 0) {
+      console.error('No email messages found for the given email address')
+      return false
+    }
+
+    const notificationsWithSubject = notificationsForEmail.filter(notification => notification.subject === subject)
+
+    if (notificationsWithSubject.length === 0) {
+      console.error(
+        `No email messages found with the given subject - subject lines: ${notificationsForEmail
+          .map(n => n.subject)
+          .join(', ')}`,
+      )
```

Comment:
Is this going to produce one really long line? Might be worth throwing in a few new lines for readability? Same below